server/views/pages/partials/printNotification/adjustmentsTable.njk

```diff
@@ -0,0 +1,28 @@
+{% macro adjustmentsTable(model) %}
+    {% set adjustmentsArray = model.generateAdjustmentsArray() %}
+    {% if adjustmentsArray.length == 0 %}
+        <p class="govuk-body govuk-!-margin-bottom-2" data-qa="adjust-desc-no">There are no adjustments.</p>
```

Comment:
Possibly untested?

Reply:
Added test case - GET /calculation/:nomsId/summary/:calculationRequestId/printNotificationSlip?fromPage=calculation should generate correct content for missing Agency Name, keyDates and Adjustments